src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -32,6 +32,8 @@
 public class PromregatorApplication {
 	
 	private static final Logger log = Logger.getLogger(PromregatorApplication.class);
+
+	public static final String SPACE_ALL_APPLICATIONS = "SPACE_ALL_APPLICATIONS";
```

Comment:
see also my comment at https://github.com/promregator/promregator/issues/35#issuecomment-384415101 regarding the "magic identifier" topic.

Reply:
Sure, using `*` character would help for a smother transition form filters based on static keys to a reg-ex like filters.